geometry_msgs/Pose pose_start
geometry_msgs/Pose pose_goal
float64            v_lin
float64            a_lin
float64[]          q_seed
---
bool success
---
float32 percent_complete
